tests/src/lease/close_position.rs

```diff
@@ -252,44 +266,65 @@ fn do_close(
     exp_change: LpnCoin,
 ) -> Addr {
     let user_balance_before: PaymentCoin = user_balance(customer, test_case);
-    let lease = lease::open_lease(test_case, DOWNPAYMENT, None);
-    let exp_lease_amount = if let StateResponse::Opened {
```

Comment:
not sure the check that `exp_lease_amount` == <balance_of_ica_after_transfer> is left in the code, please do